.github/workflows/up-version-patch.yml

```diff
@@ -0,0 +1,45 @@
+name: Bump version (patch) and create release.
+
+# Perform workflow based on human approval
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Bump version (patch) and create release.
+    runs-on: ubuntu-latest
+    # The environment is used to guarantee that a manual approval is required.
+    environment: patch_deployment
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v2
+      - name: Up version (patch)
+        run: make patch-version
+      - name: Commit version change to master
+        run: |
+          git config --global user.name "${{ github.actor }}"
+          git config --global user.email "${{ github.actor }}@users.noreply.github.com"
+          git commit -am "Automated version bump (patch)"
+          git push
+      - name: Setup release information
+        # get version number
+        run: |
+          versionName=`npm pkg get version | sed 's/"//g'`
+          export VERSION_NAME=$versionName
+          echo "::set-env name=VERSION_NAME::$VERSION_NAME"
+      - name: Extract release notes
+        id: extract_release_notes
+        uses: ffurrer2/extract-release-notes@v1
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.CLIENT_REBUILD_TOKEN_GITHUB }}
+        with:
+          tag_name: v${{ env.VERSION_NAME }}
+          release_name: ${{ env.VERSION_NAME }}
+          body: ${{ steps.extract_release_notes.outputs.release_notes }}
+          draft: false
+          prerelease: false
```

Comment:
How does this change package.json?

Reply:
We have a make command that does that for us in all the necessary files (3 actually). So we just call `make patch-version`

Reply:
PS: Internally, the command calls `npm version patch` 👍

Reply:
cool